packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -264,6 +265,18 @@ const errorsContent = (
   </EntitySwitch>
 );
 
+const pullRequestsContent = (
+  <EntitySwitch>
+    <EntitySwitch.Case if={isAzureDevOpsAvailable}>
+      <EntityAzureReposContent defaultLimit={25} />
```

Comment:
Slight nit on the naming here, perhaps `EntityAzurePullRequestsContent` for clarity?

Reply:
Thanks for taking a look. Yes, I was holding on to this naming but Marley mention another place he found it confusing and I reconsidered my stance and have adjusted this to be more clear. 👍